cmd/start.go

```diff
@@ -183,3 +191,94 @@ func initHealthServer(s *grpc.Server) {
 	gRPCHealthServer.SetServingStatus("vatz-health-status", healthpb.HealthCheckResponse_SERVING)
 	healthpb.RegisterHealthServer(s, gRPCHealthServer)
 }
+
+type PrometheusManager struct {
+	Protocol string
+	// Contains many more fields not listed in this example.
+}
+
+type PrometheusManagerCollector struct {
+	PrometheusManager *PrometheusManager
```

Comment:
Do you have to expose this property? or I recommend hide this by starting lowercased letter.

Reply:
> Commit: 

I changed it !